From dd27375873483ed7adc0064b5603a36d05dcfafa Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Wed, 23 Mar 2022 19:11:56 +0100
Subject: [PATCH 0501/1695] plat-stm32mp2: conf: support WRITE in BSEC driver

Default enable OTP write function in BSEC3 driver for platform stm32mp2

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 91c635f7fdb863097ae52ff4070fb06d88d8a153 plat-stm32mp2: conf: support WRITE in BSEC driver]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I8cf78d9b97a9ebbfe23f83283cbde2ae6bcbe3fb
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/247736
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/325722
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/354379
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index aa0f14ca6..6ea03581f 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -82,6 +82,7 @@ CFG_TEE_CORE_NB_CORE ?= 2
 CFG_STM32MP_OPP_COUNT ?= 3
 
 CFG_STM32_BSEC3 ?= y
+CFG_STM32_BSEC_WRITE ?= y
 CFG_STM32_EXTI ?= y
 CFG_STM32_FMC ?= y
 CFG_STM32_GPIO ?= y
-- 
2.47.2

